components/function-controller/internal/controllers/serverless/build_resources.go

```diff
@@ -12,6 +15,34 @@ import (
 	serverlessv1alpha1 "github.com/kyma-project/kyma/components/function-controller/pkg/apis/serverless/v1alpha1"
 )
 
+const (
+	serviceBindingUsagesTracingAnnotation = "servicebindingusages.servicecatalog.kyma-project.io/tracing-information"
```

Comment:
Do we still need to track that as we don't recreate/update Function CR (only status) in the controller?

Reply:
Ok, I see, the title confuses me as I was sure that you migrated also `UsageKind` and `ServiceBindingUsages` to Function CR. Just add a `TODO` note that it will be not needed after migration.

Reply:
After migration is also will be needed, because I retrieved in code labels from tracingAnnotations and override from `spec.podLabels` field, just to be sure of logic.

Reply:
hm... the situation is slightly different. `Kservice` was created and updated by us and we needed to be sure to not remove these labels, but `Function` is something managed by the user. As we don't control `Function` spec, we shouldn't change its values during reconciliation. Also, what if SBU change logic, or will remove first labels later annotation, we will need to adjust logic in `Function` controller. As this is the functionality of SBU leaves it for SBU, if the user removes labels or annotations, SBU will recreate it later. `Function` controller shouldn't have that logic as it is no longer needed - it is a user responsibility to not remove labels/annotations created by SBU.